### ISSUE TITLE
无法创建HugeClient

### ISSUE THREAD

**Issue by feitao1993** [apache/incubator-hugegraph#2248]
### Problem Type (问题类型)

None

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Client Version: 1.0.0 (Apache Release Version)
- springboot 3.1.1
- jdk17

### Your Question (问题描述)

在使用HugeClient.builder(url, graph).build创建hugeClient时，总是得到null

**Comment by liuxiaocs7** [comment]
[hugegraph-client](https://github.com/apache/incubator-hugegraph-toolchain/blob/master/hugegraph-client) is in [incubator-hugegraph-toolchain](https://github.com/apache/incubator-hugegraph-toolchain) repository, could we search for issues in this repository first, or move issues to this repository later~

**Comment by imbajin** [comment]
same issue already exist in toolchain, maybe check it first?


and I'll transfer the issue later, also could paste some exception log / screenshot, more info, more feedback

**Comment by feitao1993** [comment]
我没有找到类似的解决方案，能够给出一个具体的解决方法吗

**Comment by imbajin** [comment]
> 我没有找到类似的解决方案，能够给出一个具体的解决方法吗

refer：
- https://github.com/apache/hugegraph-toolchain/issues/445

**Comment by feitao1993** [comment]
参考回答进行了修改，还是出现一样的错误，具体表现如下：




pom文件的依赖如下:







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/incubator-hugegraph"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月18日(星期二) 中午11:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/incubator-hugegraph] 无法创建HugeClient (Issue #2248)





 
same issue already exist in toolchain, maybe check it first?
 
and I'll transfer the issue later, also could paste some exception log / screenshot, more info, more feedback

**Comment by z7658329** [comment]
> 参考回答进行了修改，还是出现一样的错误，具体表现如下： pom文件的依赖如下:
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "apache/incubator-hugegraph" ***@***.***&gt;; 发送时间:&nbsp;2023年7月18日(星期二) 中午11:39 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [apache/incubator-hugegraph] 无法创建HugeClient (Issue #2248) same issue already exist in toolchain, maybe check it first? and I'll transfer the issue later, also could paste some exception log / screenshot, more info, more feedback — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

i cant see your pom file from here, can you paste your pom.xml  and exception stack ?  like this issue: [445](https://github.com/apache/incubator-hugegraph-toolchain/issues/445)

**Comment by z7658329** [comment]
> > 我没有找到类似的解决方案，能够给出一个具体的解决方法吗
> 
> refer：
> 
> * [[Bug] jdk8/jdk11 springboot2.X 引用hugegraph-client1.0.0 运行报错java.lang.ClassNotFoundException: jakarta.ws.rs.ProcessingException incubator-hugegraph-toolchain#445](https://github.com/apache/incubator-hugegraph-toolchain/issues/445)
> 
> and could try common 1.0.1
> 
> @z7658329 @zyxxoo spring 3 should be fine with client？

yes 

> > 我没有找到类似的解决方案，能够给出一个具体的解决方法吗
> 
> refer：
> 
> * [[Bug] jdk8/jdk11 springboot2.X 引用hugegraph-client1.0.0 运行报错java.lang.ClassNotFoundException: jakarta.ws.rs.ProcessingException incubator-hugegraph-toolchain#445](https://github.com/apache/incubator-hugegraph-toolchain/issues/445)
> 
> and could try common 1.0.1
> 
> @z7658329 @zyxxoo spring 3 should be fine with client？

yes,  @feitao1993  if your java application use springboot3 (jdk17)  or   jdk8/jdk11 springboot2.X , you can refer to the following steps(it may works just follow step 1,  but it also fails cause the springboot version, then follow step 2):
1.upgrade your hugegraph-client version to 1.0.1
2.check your springboot version, and override jersey version in your pom.xml **(i will provide a complete config later)** :

  <properties>
          <jersery.version>3.0.3</jersey.version>
  </properties>

**Comment by z7658329** [comment]
> > 我没有找到类似的解决方案，能够给出一个具体的解决方法吗
> 
> refer：
> 
> * [[Bug] jdk8/jdk11 springboot2.X 引用hugegraph-client1.0.0 运行报错java.lang.ClassNotFoundException: jakarta.ws.rs.ProcessingException incubator-hugegraph-toolchain#445](https://github.com/apache/incubator-hugegraph-toolchain/issues/445)
> 
> and could try common 1.0.1
> 
> @z7658329 @zyxxoo spring 3 should be fine with client？


yes,  @feitao1993  if your java application use springboot3 (jdk17)  or   jdk8/jdk11 springboot2.X , you can refer to the following steps(it may works just follow step 1,  but it also fails cause the springboot version, then follow step 2):
1.upgrade your hugegraph-client version to 1.0.1
2.check your springboot version, and override jersey version in your pom.xml (i will provide a complete config later) :

  < properties>
          <jersery.version>3.0.3</jersey.version>
  </ properties>

**Comment by feitao1993** [comment]
hugegraph-client没有1.0.1版本，另外报错详细信息如下：







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/incubator-hugegraph"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月18日(星期二) 下午2:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/incubator-hugegraph] 无法创建HugeClient (Issue #2248)





   
我没有找到类似的解决方案，能够给出一个具体的解决方法吗
  
refer：
  
[Bug] jdk8/jdk11 springboot2.X 引用hugegraph-client1.0.0 运行报错java.lang.ClassNotFoundException: jakarta.ws.rs.ProcessingException&nbsp;incubator-hugegraph-toolchain#445
  
and could try common 1.0.1
 
@z7658329 @zyxxoo spring 3 should be fine with client？
  
yes,  @feitao1993  if your java application use springboot3 (jdk17)  or   jdk8/jdk11 springboot2.X , you can refer to the following steps(it may works just follow step 1,  but it also fails cause the springboot version, then follow step 2):
 1.upgrade your hugegraph-client version to 1.0.1
 2.check your springboot version, and override jersey version in your pom.xml (i will provide a complete config later) :
 
< properties&gt;
 <jersery.version&gt;3.0.3</jersey.version&gt;
 </ properties&gt;

**Comment by feitao1993** [comment]
最新的错误：







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/incubator-hugegraph"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月18日(星期二) 下午2:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/incubator-hugegraph] 无法创建HugeClient (Issue #2248)





   
我没有找到类似的解决方案，能够给出一个具体的解决方法吗
  
refer：
  
[Bug] jdk8/jdk11 springboot2.X 引用hugegraph-client1.0.0 运行报错java.lang.ClassNotFoundException: jakarta.ws.rs.ProcessingException&nbsp;incubator-hugegraph-toolchain#445
  
and could try common 1.0.1
 
@z7658329 @zyxxoo spring 3 should be fine with client？
  
yes,  @feitao1993  if your java application use springboot3 (jdk17)  or   jdk8/jdk11 springboot2.X , you can refer to the following steps(it may works just follow step 1,  but it also fails cause the springboot version, then follow step 2):
 1.upgrade your hugegraph-client version to 1.0.1
 2.check your springboot version, and override jersey version in your pom.xml (i will provide a complete config later) :
 
< properties&gt;
 <jersery.version&gt;3.0.3</jersey.version&gt;
 </ properties&gt;
